recipe/build.sh

```diff
@@ -30,8 +25,8 @@ chmod +x configure
       --with-add-fonts=${PREFIX}/fonts
```

Comment:
Do you have the motivation to try adding a fix for #23 while you're working on this?

Reply:
Let's get this merged first as it's needed for other packages like cairo.